examples/filters/sensorfilter/sensor_filter.go

```diff
@@ -90,10 +90,11 @@ func relativeChange(curr, prev map[string]interface{}, logger golog.Logger) floa
 		logger.Errorw("sensor's previous distance reading is not of type float", "prevReading", prev)
 		return 0
 	}
-	diff := currDist - prevDist
+	logger.Warnln("calculating rel change - curr: ", currDist, "; prev: ", prevDist)
```

Comment:
I'm wondering if this makes more sense at the `Info` level versus `Warn` level.

Reply:
Ignore these, they were debug lines that I later removed. Same below!